src/styles/main.css

```diff
@@ -1,11 +1,11 @@
-/* TODO: Ensure body fills entire page */
-html {
-  height: 100%;
+html, body, #root, .App {
+  height: calc(100% - 25px);
 }
 
 /* Gradient doesnt work with styled-components */
 .app-body {
-  padding-top: 65px;
+  overflow: auto;
```

Comment:
this line is the key to getting the gradient to repeat. If you remove it, the background gradient only applies to like the current viewport